src/fetchedTextProcessing.js

```diff
@@ -44,8 +45,15 @@ function processFetchedText(data) {
 
   sayFirstWordWithScreenReader(wordSpanArray[0].innerText)
 
-  currentWordSpan = wordSpanArray.shift()
+  // shift the first word span off the array and into `currentWordSpan`
+  if (wordSpanArray) {
+    currentWordSpan = wordSpanArray.shift()
+  } else {
+    currentWordSpan = null
+  }
+  // put the box around it
   currentWordSpan.classList.add("current-word")
 
+  // shift the first comparison word off the array and into `comparisonWord`
```

Comment:
Not sure why this needs changing, as it's only happening at the start of the game where there are plenty of words

Reply:
lines 48-57

Reply:
that was when I was trying to find a solution for spacing through all currentWordSpans, but the thing I added here doesn't solve that problem and is unnecessary away so will remove it

Reply:
done